jira/util/populate_db.sh

```diff
@@ -64,50 +102,68 @@ fi
 
 echo "Step2: Download DB dump"
 rm -rf ${DB_DUMP_NAME}
-wget ${DATASETS_AWS_BUCKET}/${DB_DUMP_NAME}
+wget ${DB_DUMP_URL}
 if [[ $? -ne 0 ]]; then
   echo "DB dump download failed! Pls check available disk space."
   exit 1
 fi
 
 echo "Step3: Stop Jira"
 CATALINA_PID=$(pgrep -f "catalina")
+echo "CATALINA_PID=${CATALINA_PID}"
 if [[ -z ${CATALINA_PID} ]]; then
   echo "Jira is not running"
+  sudo su -c "rm -rf ${CATALINA_PID_FILE}"
 else
   echo "Stopping Jira"
-  sudo su jira -c "echo ${CATALINA_PID} > ${CATALINA_PID_FILE}"
-  sudo su jira -c "${STOP_JIRA}"
-  sleep 5
+  sudo su -c "echo ${CATALINA_PID} > ${CATALINA_PID_FILE}"
+  sudo su -c "${SHUT_DOWN_TOMCAT}"
+  COUNTER=0
+  TIMEOUT=5
+  ATTEMPTS=20
+  while [[ "${COUNTER}" -lt "${ATTEMPTS}" ]]; do
+    if [[ -z $(pgrep -f "catalina") ]]; then
+      echo Jira is stopped
+      break
+    fi
+    echo "Waiting for Jira stop, attempt ${COUNTER} at waiting ${TIMEOUT} seconds."
+    sleep ${TIMEOUT}
+    let COUNTER++
+  done
+  if [ ${COUNTER} -eq ${ATTEMPTS} ]; then
+    echo "Jira stop was not finished in $ATTEMPTS attempts with $TIMEOUT sec timeout."
+    echo "Try to rerun script."
+    exit 1
+  fi
 fi
 
-echo "Step4: Get DB_URL"
-DB_URL=$(sudo su -c "cat ${DB_CONFIG} | grep 'jdbc:postgresql' | cut -d'/' -f3 | cut -d':' -f1")
-if [[ -z ${DB_URL} ]]; then
+echo "Step4: Get DB_HOST"
```

Comment:
Maybe in echo output, we can add more readable text, `DB Host` instead of `DB_HOST`

Reply:
Done.